templates/style.scss

```diff
@@ -140,15 +142,15 @@ div.container {
     text-align: left;
 }
 
-div.container-rustdoc {
+div.nav-container-rustdoc {
```

Comment:
When i said "move this rule to `div.nav-container-rustdoc`, i meant to use the existing block on line 265. By renaming the block like this, the div loses its `text-align: left` rule:

![image](https://user-images.githubusercontent.com/5217170/60116620-0a6a0e80-973e-11e9-80e7-0071eb90cc62.png)

Reply:
If I did that, the width of the navbar would be maximum 1200px, which wouldn't be nice. I changed something else instead. :)